integration/tests/consensus/sealing_test.go

```diff
@@ -192,12 +192,8 @@ SearchLoop:
 
 	// create the execution result for the target block
 	result := flow.ExecutionResult{
-		ExecutionResultBody: flow.ExecutionResultBody{
-			PreviousResultID: resultID,               // need genesis result
-			BlockID:          targetID,               // refer the target block
-			Chunks:           flow.ChunkList{&chunk}, // include only chunk
-		},
-		Signatures: nil,
+		PreviousResultID: resultID,               // need genesis result
+		Chunks:           flow.ChunkList{&chunk}, // include only chunk
```

Comment:
just checking it's fine to remove `BlockID:          targetID` here.

Reply:
😱 oh no, didn't mean to remove `BlockID`. Thanks for catching

Reply:
reverted accidental deletion of line `BlockID: targetID`